js/schema/plugin.json

```
@@ -26,6 +26,11 @@
           "type": "string",
           "enum": ["ask", "env", false],
           "default": "ask"
+        },
+	"defaultWritable": {
+          "description": "Set whether the fs resource contents should be treated as writable if the underlying PyFilesystem does not expose actual permissions via the access namespace",
```

```suggestion
          "description": "Fallback for determining if the fs resource can be written to. Used only if the underlying PyFilesystem does not provide this information itself",
```

make this more concise